### DISCUSSION TITLE
Custom Prometheus metrics not scraped

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#211] [Q&A]
Hi, I was testing the custom metrics exposed by lc39, using the flow-manager-client.

Using Grafana, I can see the standard metrics collected by the Prometheus server (eg. http_request_summary_seconds), but I cannot found the custom metrics, even if they are exposed via the `/-/metrics` endpoint.

I made a request to the pod and I can see the metrics exposed:
```
curl http://localhost:3000/-/metrics

  # HELP fm_client_commands_total count how many commands issued by the Flow Manager have been processed
  # TYPE fm_client_commands_total counter
  fm_client_commands_total{command="myCommand",result="SUCCESS"} 26
  ...
```

From Prometheus, I can see that the request to `/-/metrics` for my service is up and running.

Is there something else to configure? The only thing I did was enabling the metrics in the section of the console ([reference](https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/microservice-monitoring#configure-servicemonitor))

**Comment by fredmaggiowski** [comment]
Hi, did you configure the microservice in Console to have service monitoring enabled?

https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/microservice-monitoring#enable-metrics-card

**Reply by edocrippaofficial** [reply]
Is it the last line of my request?
If so, yes

I can see the default metrics collected successfully by Prometheus, but not the custom ones

**Reply by edocrippaofficial** [reply]
If I had a bad configuration in the CMS, I would expect not to be able to collect anything
Also, the servicemonitor.yml created from the console looks correct

**Reply by fredmaggiowski** [reply]
OK then it may be an issue with the exposed metrics or with the paas infrastructure; i'd suggest you to open a service request
